pkg/registry/generic/registry/store.go

```diff
@@ -40,6 +41,10 @@ import (
 	"github.com/golang/glog"
 )
 
+// EnableGarbageCollector affects the handling of Update and Delete requests. It
+// must be synced with the corresponding flag in kube-controller-manager.
+var EnableGarbageCollector bool
```

Comment:
true, though the same could be said for the existing DeleteCollectionWorkers option. Should maybe update to actually set the RESTOptions into the store?